src/test/java/spoon/test/processing/processors/ParallelProcessorTest.java

```diff
@@ -160,12 +158,12 @@ public void process(CtElement element) {
 				singleThreadCounter.incrementAndGet();
 			}
 		}).noClasspath(true).outputDirectory(folderFactory.newFolder()).buildModel();
-		for (int j = 0; j < atomicCounter.length(); j++) {
-			singleThreadCounter.set(singleThreadCounter.get() - atomicCounter.get(j));
-		}
-		assertTrue(singleThreadCounter.get() == 0);
+
+
```

Comment:
Add the following as comment so noone needs to read other tests to understand this?
// after processing both |singleThreadCounter| == sum(|atomicCounter|) must be true.

Reply:
I prefer to improve the code rather than put inline comments. I think it should be very clear from the contract of the test what the assertions are checking now, see the updated code. Wdyt?

Reply:
looks good, the comment is no longer needed.